src/cmd/manage_nsfs.js

```diff
@@ -467,7 +467,7 @@ async function update_account(data) {
  */
 async function delete_account(data) {
     await config_fs.delete_account_config_file(data);
-    return { code: ManageCLIResponse.AccountDeleted, detail: '', event_arg: { account: data.name } };
+    return { code: ManageCLIResponse.AccountDeleted, detail: `Account ${data.name} has been successfully deleted.`, event_arg: { account: data.name } };
```

Comment:
You can probably add a small line (as I assume we test the account deletion) to test the change in file: test_nc_nsfs_account_cli.test.js

Reply:
Sure.